docs/reverse_proxy.md

```diff
@@ -23,6 +23,10 @@ specification](https://matrix.org/docs/spec/server_server/latest#resolving-serve
 for more details of the algorithm used for federation connections, and
 [delegate.md](<delegate.md>) for instructions on setting up delegation.
 
+Endpoints that are part of the standardised Matrix specification are
+located under `/_matrix`, whereas endpoints specific to Synapse are
+located under `/_synapse`.
```

Comment:
we should probably say `/_synapse/client` here?

Reply:
Technically the sentence is correct, but yeah we probably don't want to encourage people to forward `/_synapse` accidentally.